components/MouseControlRotationHorizontal.cs

```diff
@@ -0,0 +1,34 @@
+using Godot;
+using static Godot.Mathf;
+
+using utilities;
+
+namespace Components;
+
+public partial class MouseControlRotationHorizontal : Node
+{
+    [Export] public bool Active = true;
+
+    [Export] public Node3D Target;
+
+    [Export] public float MouseSensitivity = 0.3f;
+
+    private float _rotationYAccumulated = 0f;
```

Comment:
you asked
> What is the point "Camera, best practices"?

What I meant by that is that horizontal rotation should be applied to body and vertical rotation need to be applied to head nodes. I wanted to just apply both on head before. That is why `MouseControlRotationHorizontal` is attached to the `PlayerCharacter` node (body) and `MouseControlRotationVertical` is attached to `Head` node.

There are additional best practices that I read in https://docs.godotengine.org/en/stable/tutorials/3d/using_transforms.html#setting-information. There they say not to reuse rotations of transform and keep these rotations outside: https://docs.godotengine.org/en/stable/tutorials/3d/using_transforms.html#setting-information

> For such cases, keep the angles and rotations outside the transform and set them every frame. Don't try to retrieve and re-use them because the transform is not meant to be used this way.

I have named these rotation accumulators `_rotationYAccumulated` and `_rotationXAccumulated`